pkg/config/config.go

```diff
@@ -82,16 +83,30 @@ var movedModules = map[string]string{
 	"community/modules/scheduler/cloud-batch-login-node": "modules/scheduler/batch-login-node",
 }
 
+// GroupName is the name of a deployment group
+type GroupName string
+
+// Validate checks that the group name is valid
+func (n GroupName) Validate() error {
+	if n == "" {
+		return errors.New(errorMessages["emptyGroupName"])
```

Comment:
nit: Previously you had suggested we start moving these to constants. I would recommend we start using the new pattern as we touch these. Would you move this particular message to a constant? Same for "duplicateGroup" message in this PR.

Reply:
Agree, but I would rather do it in separate change.